firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerformance.java

```diff
@@ -164,6 +153,7 @@ public static FirebasePerformance getInstance() {
    * @param gaugeManager The GaugeManager instance.
    */
   @VisibleForTesting
+  @Inject
```

Comment:
Since now we've changed `FirebasePerformance` object to be dependency injected don't we want to update the [corresponding tests](https://github.com/firebase/firebase-android-sdk/blob/6270c03055591f55b4206c06373a7184012952f7/firebase-perf/src/test/java/com/google/firebase/perf/FirebasePerformanceTest.java#L565-L573)?

Reply:
Dependency injection does not change the way we create objects and we did not modify any existing constructors. Also, [the dagger dev site](https://dagger.dev/dev-guide/testing.html) says "You don’t have to use Dagger for single-class unit tests". Therefore, we don't have to update the tests. 
@visumickey What do you think?

Reply:
Maybe fine for now (since the change scope is very limited currently) but I assume as we expand on this we would want to change the way how we call constructors from the tests.